packages/lodestar/src/chain/blocks/importBlock.ts

```diff
@@ -71,12 +71,10 @@ export type ImportBlockModules = {
  * - head_tracker.register_block(block_root, parent_root, slot)
  * - Send events after everything is done
  */
-export async function importBlock(chain: ImportBlockModules, fullyVerifiedBlock: FullyVerifiedBlock): Promise<void> {
+export async function importBlock(modules: ImportBlockModules, fullyVerifiedBlock: FullyVerifiedBlock): Promise<void> {
```

Comment:
Why rename `chain` to `modules`? If you believe this is an important change it should be done in another PR and keep this diff as small as possible

Reply:
> Why rename chain to modules?

The type says `ImportBlockModules` with modules in it, and coming first to the code base, when I saw `chain` I initially thought this was referring to `IBeaconChain`.

> If you believe this is an important change it should be done in another PR and keep this diff as small as possible

Since this rename is contained to a file, I thought why not just change it here, instead of creating another PR just for it. But I can definitely revert and make another PR if that is the preference.

Reply:
> I thought why not just change it here

doppelganger is a critical feature. A bigger diff correlates with inferior review quality. So for the safety of the feature the diff must be minimal

> But I can definitely revert and make another PR if that is the preference.

please :pray: